library/ix-dev/community/audiobookshelf/item.yaml

```diff
@@ -0,0 +1,5 @@
+icon_url: https://cdn.jsdelivr.net/gh/walkxcode/dashboard-icons/svg/audiobookshelf.svg
```

Comment:
```suggestion
icon_url: https://media.sys.truenas.net/apps/audiobookshelf/icons/icon.svg
screenshots: 
  - https://media.sys.truenas.net/apps/audiobookshelf/screenshots/screenshot1.png
  - https://media.sys.truenas.net/apps/audiobookshelf/screenshots/screenshot2.png
  - https://media.sys.truenas.net/apps/audiobookshelf/screenshots/screenshot3.png
  - https://media.sys.truenas.net/apps/audiobookshelf/screenshots/screenshot4.png
```

Reply:
Awesome!  Thank you for adding the svg and screenshots to iX site!

`screenshot4.png` was not found.  Did you want to re-upload or have me keep the first 3?

Reply:
Updated the link! It was an jpg!